ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -367,7 +367,6 @@ Item {
             onLaunchBuyCryptoModal: {
                 d.buyFormData.selectedWalletAddress = d.getSelectedOrFirstNonWatchedAddress()
                 d.buyFormData.selectedNetworkChainId = StatusQUtils.ModelUtils.getByKey(RootStore.filteredFlatModel, "layer", 1, "chainId")
-                d.buyFormData.selectedTokenKey = Constants.ethToken
```

Comment:
The idea here is to be able distinguish between explicitly buying ETH or just any token; we can make the fallback later in the BuyCryptoModal...